docs/tutorials/sinks/1_basic_sink.md

```diff
@@ -0,0 +1,308 @@
+Let's write a basic sink for Vector.
+
+Create a new rust module in `src/sinks/basic.rs`.
+
+# Imports
+
+Let's setup all the imports we will need for the tutorial:
+
+```rust
```

Comment:
Maybe too much for the basic tutorial, but should we mention what each of the imports is "for"?

Reply:
I think that would just be annoying to do here, but hopefully it is explained through out as each import is used.